core/src/avm1/object/script_object.rs

```diff
@@ -291,18 +282,14 @@ impl<'gc> TObject<'gc> for ScriptObject<'gc> {
     fn call_setter(
         &self,
         name: &str,
-        value: Value<'gc>,
+        _value: Value<'gc>,
```

Comment:
Should we remove this param since it is unused? I'm also wondering if we can ditch `call_setter` in favor of inspecting the property directly in `TObject::set`.